doc/api/net.md

```diff
@@ -100,47 +169,65 @@ deprecated: v0.9.7
 
     Stability: 0 - Deprecated: Use [`server.getConnections()`][] instead.
 
-The number of concurrent connections on the server.
+A property indicating the number of concurrent connections on the server.
 
-This becomes `null` when sending a socket to a child with
+The value is `null` when a socket is sent to a child using
 [`child_process.fork()`][]. To poll forks and get current number of active
-connections use asynchronous `server.getConnections` instead.
+connections use the `server.getConnections()` method instead.
```

Comment:
Isn't it .send() we use to send sockets to child processes over IPC? This makes it sound like .fork() does the `fork(2)` process duplication, where sockets are also duplicated. Also, if you do spawn with an IPC channel, you can also use .send() to send a socket. I don't think this text was correct.